### PR TITLE
Feature/ui my profile

### PR DESCRIPTION
**Pull request by pkvarnwal**

### CODE REVIEW

File: app/src/main/java/com/eqdepot/adapters/MyAccountAdapter.java [hunk 1]
```diff
@@ -60,7 +62,8 @@ private void bindView(int position) {
 
         @OnClick(R.id.relative_layout_my_account)
         void ItemTapped() {
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/adapters/MyAccountAdapter.java]: please name it onItemTapped()

Comment (pkvarnwal) [reply app/src/main/java/com/eqdepot/adapters/MyAccountAdapter.java]: Done

**Comment by ashutoshsingh33** [comment]
@aashishdhawan  Please review.

File: app/src/main/java/com/eqdepot/activities/MyProfileActivity.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_my_profile);
+        ButterKnife.bind(this);
+        textViewHeaderTitle.setText(getResources().getString(R.string.my_profile));
+        setFont();
+    }
+
+    private void setFont() {
+        FontHelper.setFontFace(FontHelper.FontType.FONT_MEDIUM, textViewName, textViewTitle, textViewStreet, textViewCity,
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/activities/MyProfileActivity.java]: 👍

**Pull request merged**